web/lib/src/helpers/lists.dart

```diff
@@ -103,10 +103,22 @@ class JSLiveNodeListWrapper<P extends Node, T extends JSObject, U extends Node>
     parentNode.appendChild(element);
   }
 
+  @override
+  bool remove(Object? element) {
+    // ignore: invalid_runtime_check_with_js_interop_types
```

Comment:
Let's add a todo on me to migrate this to `isJSAny` once we have it.

Reply:
done